web/tests/api-spec.js

```diff
@@ -4,13 +4,15 @@ describe('api service', function () {
         fixture.setBase('tests/api-description');
 
         inject(function ($http) {
-            const MOCK_PROJECT_SETTINGS = {
-                API_ROOT: 'localhost:8000'
+            const mockProjectConfig = {
+                getSettings:() => ({
+                    apiRoot: 'localhost:8000'
+                })
```

Comment:
In general I am against arrow functions for the simple reason of readability.

```
const x = function () {
 return {
   apiRoot: 'blah';
 }
};
```

is much more human readable than:

```
const x() => ({
  apiRoot: 'localhost:8000'
});
```

What advantage does the latter give? The only one I see is shorter code, which is not an issue when the final thing will be minified for production. The readability by other devs is much more important in this case and I think should take priority over shorter code.

I know this is syntax allowed by the spec, but I don't think we should use it for the same reason we try to avoid other language shortcuts. It makes it harder to human read for anyone not immediately familiar with the syntax. New developers, people coming from other languages or even people just not familiar with this bit of code will understand the former much more easily.

I can see a small argument for using them when you really need to bind `this` (although I'd have preferred they implement a `scopedFunction()` or even using `bind()` instead of the arrows), but otherwise I don't think we should use them generally.